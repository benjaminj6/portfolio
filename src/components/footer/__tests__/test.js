/* eslint-env jest */
import { h } from 'preact'
import { shallow } from 'preact-render-spy'

import Footer from '../'

test('Footer snap', () => {
  const tree = shallow(<Footer />)
  expect(tree).toMatchSnapshot()
})
