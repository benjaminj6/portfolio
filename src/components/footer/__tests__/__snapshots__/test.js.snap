// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Footer snap 1`] = `
preact-render-spy (1 nodes)
-------
<footer class="jsx-3347494647 jsx-3347494647">
  <ul class="jsx-3347494647 jsx-3347494647">
    <li class="jsx-3347494647 jsx-3347494647">
      <SocialMediaLink
        url="https://awesome-test.com"
        icon={[Function icon]}
      >
      </SocialMediaLink>
    </li>
  </ul>
  <p class="jsx-3347494647">Â© 2017 Benjamin Johnson</p>
  <JSXStyle
    styleId="3347494647"
    css="footer.jsx-3347494647{background-color:var(--gray-lightest);padding:1rem;}ul.jsx-3347494647{display:-webkit-box;display:-webkit-flex;display:-ms-flexbox;display:flex;-webkit-box-pack:center;-webkit-justify-content:center;-ms-flex-pack:center;justify-content:center;}p.jsx-3347494647{text-align:center;font-size:0.75rem;}"
  >
  </JSXStyle>
</footer>

`;
